skada/metrics.py

```diff
@@ -171,7 +171,10 @@ def _score(self, estimator, X, y, sample_domain=None, **params):
         ws = self.weight_estimator_source_.score_samples(X_source)
         wt = self.weight_estimator_target_.score_samples(X_source)
         weights = np.exp(wt - ws)
-        weights /= weights.sum()
+
+        if weights.sum() != 0:
```

Comment:
How can we get 0 if weights are exp? 🤔

Reply:
I think its due to python precision, I've checked the value manually for an exp and I had at some point `exp(-20_000) = 0`

Reply:
so you should probably use logsumexp that is numrically stable there (susing a concatenation of ws wt)

Reply:
So, if we got all weights 0., the following estimator should fail anyways. We should log warning here, that it's the reweighter who's to blame for the failure.

Reply:
I tried with logsumexp but we have the same issue. I propose logging a warning as @kachayev suggested + using uniform weights instead